docs/src/app/pages/Form/Checkbox/ExampleErrorMessage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Checkbox } from 'react-lds';
+
+const ExampleErrorMessage = () =>
+  <div>
+    <Checkbox
+      id="checkbox-input-1"
+      label="Checkbox Label"
+      error="Something is wrong"
+      errorMessage={false}
```

Comment:
LGTM! Just a detail: I would call the prop `showErrorMessage` or `hideErrorMessage` to make its purpose clearer.

Reply:
Makes sense. I renamed it to hideErrorMessage so it can be used as a boolean flag like `<Input ... hideErrorMessage />`